src/main/java/nl/ordina/jobcrawler/service/LogService.java

```diff
@@ -0,0 +1,19 @@
+package nl.ordina.jobcrawler.service;
+
+
+// Added to assert that logging is called in certain situations
+// for test purposes
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class LogService {
+
+    public void logInfo(String aMessage) {
+        log.info(aMessage);
+    }
+
+    public void logError(String aMessage) {
+        log.error(aMessage);
+    }
+}
```

Comment:
This is a good idea, to have our own streamlined logging service. However to be consistent we should use it for all logging or not using it. To be discussed in the meeting.

Reply:
the other code has no direct use for it. It has been introduced here because some tests depend on log mocking